build-system/test-configs/forbidden-terms.js

```diff
@@ -746,15 +746,11 @@ const forbiddenTermsGlobal = {
       'extensions/amp-stream-gallery/1.0/storybook/Basic.amp.js',
       'extensions/amp-stream-gallery/1.0/storybook/Basic.js',
       'extensions/amp-timeago/1.0/storybook/Basic.js',
-      'extensions/amp-twitter/0.1/storybook/Basic.amp.js',
-      'extensions/amp-twitter/1.0/storybook/Basic.amp.js',
-      'extensions/amp-twitter/1.0/storybook/Basic.js',
       'extensions/amp-video/1.0/storybook/Basic.amp.js',
       'extensions/amp-video/1.0/storybook/Basic.js',
       'extensions/amp-video-iframe/1.0/storybook/Basic.amp.js',
       'extensions/amp-vimeo/1.0/storybook/Basic.amp.js',
       'extensions/amp-vimeo/1.0/storybook/Basic.js',
-      'extensions/amp-wordpress-embed/1.0/storybook/Basic.amp.js',
```

Comment:
@rsimha Added you as reviewer due to items to remove in this allowlist. 

Since we'll have to make 5-7 changes to the list in the next few weeks, could `wg-components` (or someone from the group) be allowed to approve this file temporarily?

Reply:
Absolutely. Might be worth making `wg-components` an owner of this file.

Reply:
Thanks!